tool/teleport/testenv/test_server.go

```diff
@@ -46,8 +46,6 @@ import (
 	"github.com/gravitational/teleport/tool/teleport/common"
 )
 
-var ports utils.PortList
```

Comment:
Are there still uses of this, or can we remove PortList and GetFreeTCPPorts?

Reply:
It is still used in `tsh` and integration tests.